src/guides/v2.3/extension-dev-guide/routing.md

```diff
@@ -350,6 +350,73 @@ As a result, by accessing the `http://site.com/learning` route, the `http://site
 
 ![Routing Result]({{ site.baseurl }}/common/images/routing-result.png)
 
+## Declaring the new custom noRoute processor
+
+Magento allows to add a custom noRoute processor.
```

Comment:
Please, provide a sentence or two about the use case of a custom noRoute processor. So, why a developer might need a custom noRoute processor?

Thank you.